src/driver/postgres/PostgresDriver.ts

```diff
@@ -1388,6 +1388,27 @@ export class PostgresDriver implements Driver {
         return false
     }
 
+    /**
+     * Returns true if the current version of Postgres supports nulls not distinct.
+     *
+     * Since this feature is only implemented in PostgreSQL 15 and above, other
+     * versions of Postgres will throw an error when using nullsNotDistinct unless
+     * throwOnUnsupported is set to false.
+     */
+    isNullsNotDistinctSupported({
+        throwOnUnsupported = true,
```

Comment:
I would remove this parameter and gracefully handle it in each situation (i.e. without throwing).